audit.go

```diff
@@ -339,10 +339,18 @@ func main() {
 	}
 
 	nlClient, err := NewNetlinkClient(config.GetInt("socket_buffer.receive"))
+
 	if err != nil {
 		el.Fatal(err)
 	}
 
+	dnstapClient, err := NewDnsTapClient(config.GetString("dnstap.socket"))
```

Comment:
should this block be within a `if config.GetString("dnstap.socket") != ""`?

Reply:
good call, thanks for that!